src/pylero/base_polarion.py

```diff
@@ -341,6 +341,7 @@ def _convert_obj_fields_to_polarion(cls, fields=[]):
                     else cls._cls_suds_map[x]["field_name"],
                 )
                 for x in fields
+                if x in cls._cls_suds_map.keys()
```

Comment:
why do we need an explicit check?

Reply:
Because (for me at least) not every class contains the same custom fields. 
As explained in my issue https://github.com/RedHatQE/pylero/issues/136, this is where I get a key error!